File: backend/src/main/java/com/davnig/invoicemanagementapi/model/dto/TaxableSubjectSummary.java
package com.davnig.invoicemanagementapi.model.dto;

import com.davnig.invoicemanagementapi.model.entity.TaxableSubject;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class TaxableSubjectSummary {

    private String piva;
    private String taxCode;

    public TaxableSubjectSummary(TaxableSubject entity) {
        this.piva = entity.getPiva();
        this.taxCode = entity.getTaxCode();
    }

}

File: backend/src/main/kotlin/com/davnig/invoicemanagementapi/model/dto/TaxableSubjectSummary.kt
package com.davnig.invoicemanagementapi.model.dto

import com.davnig.invoicemanagementapi.model.entity.TaxableSubject

class TaxableSubjectSummary {

    var piva: String? = null
    var taxCode: String? = null

    constructor(entity: TaxableSubject) {
        piva = entity.piva
        taxCode = entity.taxCode
    }

}